SPECS/klipper-lb/klipper-lb.spec

```diff
@@ -0,0 +1,38 @@
+Summary:        This is the klipper runtime image for the integrated service load balancer
+Name:           klipper-lb
+Version:        0.3.5
+Release:        1%{?dist}
+License:        ASL 2.0
+URL:            https://github.com/k3s-io/klipper-lb
+Group:          Applications/Text
+Vendor:         Microsoft
+Distribution:   Mariner
+Source0:        https://github.com/k3s-io/%{name}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+#Note that the source file should be renamed to the format {name}-%{version}.tar.gz
```

Comment:
Nit-pick: usually we add comments regarding a tag above it and use double `%` characters in the comments, so that the spec parsing tooling won't print warnings:

```suggestion
# Note that the source file should be renamed to the format '%%{name}-%%{version}.tar.gz'
Source0:        https://github.com/k3s-io/%{name}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```